### PR TITLE
regenerated from latest master

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
cc @lorenzobettini

File: greetings-maven/2.20.0-J11/org.xtext.example.mydsl.parent/pom.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 	<properties>
 		<xtextVersion>2.20.0-SNAPSHOT</xtextVersion>
+		<mwe2Version>2.11.1</mwe2Version>
```
Comment (cdietrich) [inline greetings-maven/2.20.0-J11/org.xtext.example.mydsl.parent/pom.xml]: @lorenzobettini, do i see correct the var is not used in pure maven.
do we want this?

**Comment by LorenzoBettini** [comment]
Sorry I missed this one. That's solved, isn't it?

**Comment by cdietrich** [comment]
Yes regenerated with your second change

**Pull request merged**